packages/shared/src/hooks/useChangelog.ts

```diff
@@ -13,15 +12,12 @@ type UseChangelog = {
 
 export function useChangelog(): UseChangelog {
   const { alerts, updateAlerts } = useContext(AlertContext);
-  const { user } = useContext(AuthContext);
   const { sidebarRendered } = useSidebarRendered();
 
   const { data: latestPost } = useQuery(
-    ['changelog', 'latest-post', { loggedIn: !!user?.id }] as const,
-    async ({ queryKey }) => {
-      const [, , variables] = queryKey;
-
-      return getLatestChangelogPost(variables.loggedIn);
+    ['changelog', 'latest-post'] as const,
```

Comment:
The `as const` is not needed here I don't think?

Reply:
It does not anymore, it was required when queryKey had dynamic parts.